### PR TITLE
Implement aten::lift_fresh_copy* ops | feat(torchlib)

### PR DESCRIPTION
**Pull request by wschin**

### CODE REVIEW

**Review by justinchuby — APPROVED** [review]

**Comment by wschin** [comment]
@justinchuby, is my 2nd commit for adding test ok?

**Comment by justinchuby** [comment]
It will likely fail because we need the first name "clone" to be unique in the list. I wouldn't bother adding a test since it's just identity. If we really want we should create an opinfo to use the proper (torch.ops.aten.left_fresh_copy) op.

**Comment by wschin** [comment]
Ok. Let me revert.

**Comment by justinchuby** [comment]
If we **really** just want to reuse clone, follow this: https://github.com/microsoft/onnxscript/blob/18409c15a8d86c41cbe91753b45234832d1df2ce/onnxscript/tests/function_libs/torch_lib/ops_test_data.py#L1794

ops_test_common.duplicate_opinfo(OPS_DB, "clone", ("ops.aten.left_fresh_copy"))

**Comment by wschin** [comment]
Thanks. I will use the duplicate_opinfo.

**Comment by wschin** [comment]
Is this the only change I need?
```
ops_test_common.duplicate_opinfo(OPS_DB, "clone", ("clone", "lift_fresh_copy"))
```

**Comment by justinchuby** [comment]
Do 

```diff
- ops_test_common.duplicate_opinfo(OPS_DB, "clone", ("clone", "lift_fresh_copy"))
+ ops_test_common.duplicate_opinfo(OPS_DB, "clone", ("lift_fresh_copy",))
```

then use `lift_fresh_copy` in the `TorchOpInfo` entry.

**Comment by wschin** [comment]
How about [now](https://github.com/microsoft/onnxscript/pull/928/commits/6c1bdf313b83bee927583a8c998e70912766a3a9)?

File: onnxscript/tests/function_libs/torch_lib/ops_test_data.py [hunk 1]
```diff
@@ -578,6 +578,7 @@ def _where_input_wrangler(
         reason="fixme: ORT has memory errors. https://github.com/microsoft/onnxruntime/issues/16492",
     ),
     TorchLibOpInfo("clone", core_ops.aten_clone),
+    TorchLibOpInfo("lift_fresh_copy", core_ops.aten_lift_fresh_copy),
```
Comment (justinchuby) [inline onnxscript/tests/function_libs/torch_lib/ops_test_data.py:581]: nit: sort alphabetically

**Review by justinchuby — APPROVED** [review]
LGTM!

File: onnxscript/tests/function_libs/torch_lib/ops_test_data.py (hunk 1 above)
Comment (wschin) [reply onnxscript/tests/function_libs/torch_lib/ops_test_data.py]: Ok.

**Pull request merged**